### ISSUE TITLE
render mathml

### ISSUE THREAD

**Issue by richcanvas** [KaTeX/KaTeX#3852]
can Katex.render(mathml) directly?

**Comment by grigoriy-reshetniak** [comment]
Yes, you can use `output: "mathml"` in [options](https://katex.org/docs/options).

Also it worth to check [Temml](https://temml.org/).
